dvc/command/diff.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import unicode_literals
+
+import humanize
+import inflect
+
+
+import dvc.logger as logger
+from dvc.exceptions import DvcException
+from dvc.command.base import CmdBase
+from dvc.utils.collections import compact
+import dvc.repo.diff as diff
+
+
+class CmdDiff(CmdBase):
+    def _print_size(self, size):
+        if size < 0:
+            change = "decreased by {}"
+        elif size > 0:
+            change = "increased by {}"
+        else:
+            change = "not changed"
+        natur_size = humanize.naturalsize(abs(size))
+        return change.format(natur_size)
+
+    def _show(self, diff_dct):
```

Comment:
Could you also add a test for CLI output please? If you take a look at the coverage of this file, you could see that `_show()` is not tested at all.